package.json

```diff
@@ -5,9 +5,27 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "files": [
+    "/bin",
     "/lib"
   ],
+  "bin": {
+    "strong-config": "./bin/run"
+  },
+  "oclif": {
+    "commands": "./lib/cli/commands",
+    "bin": "strong-config",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-warn-if-update-available",
+      "@oclif/plugin-not-found"
```

Comment:
did you have a look at the [autoupdater plugin](https://oclif.io/docs/releasing.html#autoupdater)? could be useful